pytorch_blade/torch_blade/mlir/hash.py

```diff
@@ -13,14 +13,9 @@
 import numpy as np
 import hashlib
 from torch_blade._torch_blade import _backends
+from torch_blade import hash_data, hash_combine
 HASH_SEED = 1
 
-def data_hash(value: str) -> int:
```

Comment:
nit: removed for performance reason?

Reply:
hashlib.sha256 returns a 64  character hash string that is too long and increased infinity after `hash_combine`.